modules/ROOT/pages/environments.adoc

```diff
@@ -54,10 +60,32 @@ To view the client ID and secret for your environment:
 
 . Log in to the Anypoint Platform as an administrator.
 . Go to Access Management.
-. From the left-hand menu, click *Environments*.
+. In the menu on the left, click *Environments*.
 . Click the name of the environment whose client ID/secret you want to view.
 
 [NOTE]
 --
 To change the client ID or client secret of an environment, you must contact your customer support representative.
 --
+
+== Add an IdP to an Existing Environment
+
+If your organization administrator provides multiple xref:managing-api-clients.adoc[identity providers], or IdPs, for your master organization, you can add them to existing environments as client providers. When you create a new API in an environment that has an IdP, the API receives the default client provider.
```

Comment:
Do we refer to 'master' organizations elsewhere? I thought organizations were always the master org in docs, and business groups non-master organizations.

Reply:
"receive" a default client provider isn't exactly clear to me, here. I would attempt to reduce the mentions we do to API Manager and its logic, and say that it associates an environment with an IDP, enabling it to be used for API Management.

Reply:
```suggestion
If your organization administrator provides multiple xref:managing-api-clients.adoc[identity providers], or IdPs, for your organization, you can add them to existing environments as client providers.
```

Reply:
Going to leave that API detail to API Manager as we discussed.

Reply:
```suggestion
If your organization administrator provides multiple xref:managing-api-clients.adoc[identity providers], or IdPs, for your organization, you can add them to existing environments as client providers.
```

Reply:
Going to leave that bit to API Manager doc as we discussed. Removing instances of "master" in "master organization" since it's redundant. I think we assume master org unless we call out a BG specifically.